### ISSUE TITLE
How to run this on all subfolders?

### ISSUE THREAD

**Issue by selvavm** [jepperaskdk/pydoctest#33]
Hi,

Is it possible to run this from a folder and look into all the py files in the sub-folders?

Something like below,
```
pydoctest --parser google --include_paths "**/*.py"
```

**Comment by jepperaskdk** [comment]
Hi @selvavm,

I haven't yet added `include_paths` as a CLI argument (it should be added), but if you create a pydoctest.json in your root directory that looks like this:
```
{
    "include_paths": [ "*.py", "**/*.py" ]
}
```
Then it should analyze current directory and subdirectories. I just did some testing, and it doesn't seem like recursive discovery is possible currently. pydoctest uses `glob` which doesn't seem to support `**` directly (but has a `recursive=True` parameter. There's a `glob2` module which seems to support it. Let me know if you need recursive discovery, then I'll look into it.

**Comment by selvavm** [comment]
Thanks @jepperaskdk. As of now, I do not need recursive but having it can be best fit for CI

**Comment by jepperaskdk** [comment]
I started working on recursive globbing (and exclude paths) in this PR: [Support for exclude paths and recursive glob](https://github.com/jepperaskdk/pydoctest/pull/35)

For the CLI commands, do you have suggestions for the format of the patterns?
Could be like this, where the list is quoted and comma separated:
```
pydoctest --include-paths "*.py, **/*.py" --exclude-paths "**/__init__.py, **/setup.py"
```

**Comment by jepperaskdk** [comment]
@selvavm I have now released version 0.1.17, which adds:
- Support for recursive globbing with `**` 
- Support for exclude_paths to exclude files based on patterns
- CLI support for both `--include-paths` and `--exclude-paths`

Specific to your use-case, it now also defaults to the recursive glob `**/*.py`, so you may not need the cli-argument at all.
Let me know if something is not as expected!
